### DISCUSSION TITLE
Helping The Beginners Click Effect

### DISCUSSION THREAD

**Discussion by TheBeginnerOne** [twbs/bootstrap#36341] [Q&A] [answered]
Hi Guys, I am a Really Beginner At HTML, CSS, and JS, I Would Like to Know How to Create a Click Effect (I Would Like to Create a Zoom Effect in an Image by Clicking, When I Click in The Image it Gets the Zoom and When I Click out The Image the Zoom Goes out., if Possible using Only HTML and CSS If Not There is No Problem Use JS.
Please Help Me.
I Would Like the Effect to be Like in This Site Images.
https://guimattos.com.br/2021-baroneza-11

**Comment by gerardnico** [comment, marked as answer]
This is called a lightbox.

You can do it with bootstrap via the modal where the image is the content.

**Reply by TheBeginnerOne** [reply]
Thanks
